jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/CustomDateTimeFormatIT.java

```diff
@@ -0,0 +1,182 @@
+    private Object javaValue;
+
+    public CustomDateTimeFormatIT(String propertyName, Boolean config_isFormatDateTime, Boolean isAnnotated, String format, String timezone, Object jsonValue, Object javaValue) {
+        this.propertyName = propertyName;
+        this.config_isFormatDateTime = config_isFormatDateTime;
+        this.isAnnotated = isAnnotated;
+        this.format = format;
+        this.timezone = timezone;
+        this.jsonValue = jsonValue;
+        this.javaValue = javaValue;
+    }
+
+    /**
+     * We are going to generate the same class twice:
+     * Once with the configuration option formatDateTime set to TRUE
+     * Once with the configuration option formatDateTime set to FALSE
+     * 
+     * @throws ClassNotFoundException
+     * @throws IOException
+     */
+    @BeforeClass
+    public static void generateClasses() throws ClassNotFoundException, IOException {
+        Map<String, Object> configValues = new HashMap<String, Object>();
+        
+        configValues.put("formatDateTime", Boolean.TRUE);
+        classSchemaRule.generate("/schema/format/customDateTimeFormat.json", "com.example.config_true", configValues);
+        
+        configValues.put("formatDateTime", Boolean.FALSE);
+        classSchemaRule.generate("/schema/format/customDateTimeFormat.json", "com.example.config_false", configValues);
+        
+        ClassLoader loader = classSchemaRule.compile();
+        
+        // Class generated when formatDateTime is set to TRUE in configuration
+        classWhenConfigIsTrue = loader.loadClass("com.example.config_true.CustomDateTimeFormat");
+        // Class generated when formatDateTime is set to FALSE in configuration
+        classWhenConfigIsFalse = loader.loadClass("com.example.config_false.CustomDateTimeFormat");
+    }
+    
+    /**
+     * Test whether the annotation is supposed to be generated.
+     * If it is supposed to be generated, test if the pattern and timezone attributes are as expected.
+     * 
+     * @throws NoSuchFieldException
+     * @throws SecurityException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     */
+    @Test
+    public void formatValueProducesExpectedAnnotation() throws NoSuchFieldException, SecurityException, InstantiationException, IllegalAccessException {
+        Field field;
+        if (config_isFormatDateTime) {
+            field = classWhenConfigIsTrue.getDeclaredField(propertyName);
+        }
+        else {
+            field = classWhenConfigIsFalse.getDeclaredField(propertyName);
+        }
+        
+        if (isAnnotated) {
+            JsonFormat annotation = field.getAnnotation(JsonFormat.class);
+            
+            assertThat(annotation, notNullValue());
+            // Assert that the patterns match
+            assertEquals(format, annotation.pattern());
+            // Assert that the timezones match
+            assertEquals(timezone, annotation.timezone());
+        }
```

Comment:
This project uses the following style for if:

```java
if (...) {

} else {

}
```

Could you follow this? Thanks